charts/chronicle/values.yaml

```diff
@@ -142,6 +144,11 @@ test:
       repository: blockchaintp/chronicle-helm-api-test-amd64
       ## @md | `test.api.image.tag` | the image tag | latest |
       tag: BTP2.1.0-0.7.3
+  auth:
+    ## @md | `test.auth.enabled` | true to enable testing with authorization header | true |
+    enabled: true
```

Comment:
is there a time when you want this to be true, and yet have devIdProvider==false, no URLs provided, and auth.required==false?